.travis.yml

```diff
@@ -2,17 +2,24 @@ group: travis_latest
 language: python
 cache: pip
 python:
-    - 2.7
+- 3.7
 install:
-    #- pip install -r requirements.txt
-    - pip install flake8  # pytest  # add another testing frameworks later
+- pip install -r monkey/monkey_island/requirements.txt
+- pip install flake8 pytest dlint
```

Comment:
pylint is also a good linter if we want as much warnings as possible :D

Reply:
Not sure "as many warnings as possible" is the goal. Pylint is REALLY whiny... And it'll take some configuration before it works properly. So maybe in future versions. For now, I think `dlint` and `flake8` is more than enough. 

Also, I'm probing into adding [SonarQube](https://www.sonarqube.org) support in future.